cubeseed_login/components/create-invoice.tsx

```diff
@@ -1,15 +1,15 @@
-"use client";
+"use client"
 
-import Image from "next/image";
-import { useState } from "react";
-import billto from "@cs/public/bill-to.png";
-import payto from "@cs/public/pay-to.png";
-import Input from "./pieces/input";
-import Textarea from "./pieces/textarea";
-import Table from "./pieces/table";
-import CreateSucess from "./create-success";
-import { BillType, OrderType, SummaryType } from "@cs/types";
-import { DashboardHeading } from "./header";
+import Image from "next/image"
+import { useState } from "react"
+import billto from "@cs/public/bill-to.png"
+import payto from "@cs/public/pay-to.png"
+import Input from "./pieces/input"
+import Textarea from "./pieces/textarea"
+import Table from "./pieces/table"
+import CreateSucess from "./create-success"
+import { BillType, OrderType, SummaryType } from "@cs/types"
+import { DashboardHeading } from "./header"
```

Comment:
Some of these changes could have been made in another PR or write clear commit messages for them so it's easily trackable.